dotnet/samples/AgentSyntaxExamples/RepoUtils/ObjectExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+
+namespace RepoUtils;
+
+public static class ObjectExtensions
```

Comment:
I pruned out a bunch of stuff agents will be unlikely to use, but I'd like to reserve the ability to use this extension in the future.  (Was wondering if this might spark discussion on a code sharing pattern for *SyntaxExamples.)

Reply:
Add the code when you need it.

As I mentioned in another comment the current pattern for syntax examples is being replaced so all of this code will need to be redone. I'd recommend keeping the amount of code in this project minimal to avoid building up technical debt.